### PR TITLE
Add 1.26 conformance certified image

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*
Add 1.26 conformance certified image

Conformance confirmation: https://github.com/cncf/k8s-conformance/pull/2531

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ Source: https://github.com/cncf/artwork/tree/master/projects/kubernetes/certifie
 -->
 [<img src="docs/static/images/certified-kubernetes-1.23-color.svg" height=150>](https://github.com/cncf/k8s-conformance/pull/2109)
```
Comment (TerryHowe) [inline README.md:23]:
```suggestion
```

Comment (taneyland) [reply README.md:23]: Why delete this? We still support 1.23

Comment (TerryHowe) [reply README.md:23]: Check the rules on k8s-conformance. We probably cannot post that by the agreement. I used to get email notifications on when the cerifications expire.

Comment (taneyland) [reply README.md]: Ah I see. Updated the PR. Thanks!

**Pull request merged**